scripts/plot_network.py

```diff
@@ -17,7 +17,8 @@
 
 import logging
 logger = logging.getLogger(__name__)
-from _helpers import load_network, aggregate_p, aggregate_costs, configure_logging
+from _helpers import (load_network, aggregate_p, aggregate_costs,
+                      configure_logging, mocksnakemake)
```

Comment:
```suggestion
                      configure_logging)
```

to stay consistent with the other files.